portal/settings.py

```diff
@@ -13,6 +13,11 @@
 import os
 from pathlib import Path
 
+from dotenv import load_dotenv
+
+load_dotenv()
```

Comment:
The error with the linter is because there is a function call here, followed by an import statement below (`import dj_database_url`).
All the imports should be done at the top of the file, so the function call (`load_dotenv()`) must be moved to after all the imports.

But I wonder if you should remove the dotenv completely. Why is it needed?